Sources/App/main.swift

```diff
@@ -8,9 +8,26 @@
 
 import Guaka
 
-setup()
 
-root.add(flag: Flag(longName: "togge", value: false, shortName: "t", inheritable: false))
-root.add(flag: Flag(longName: "root", value: 1, shortName: "r", inheritable: false))
+[
+  Flag(longName: "foo", value: "", inheritable: true),
+  Flag(longName: "remote", value: true, inheritable: true),
+  Flag(longName: "bar", value: "", inheritable: false),
+  Flag(longName: "xx", value: true, inheritable: false),
+  ].forEach { initCommand.add(flag: $0) }
```

Comment:
The init will be generated from the cli, the flags will be mostly added by the dev (that is inline with cobra as it does not have the ability to add flags)

Having said that, the constructor accepts a `[Flag]` so we can add them there!